client/post-editor/copy-post/accordion.jsx

```diff
@@ -0,0 +1,116 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+import cloneDeep from 'lodash/cloneDeep';
+import isEmpty from 'lodash/isEmpty';
+
+/**
+ * Internal dependencies
+ */
+import Accordion from 'components/accordion';
+import AccordionSection from 'components/accordion/section';
+import Button from 'components/button';
+import Gridicon from 'components/gridicon';
+import PostActions from 'lib/posts/actions';
+import PostSelector from 'my-sites/post-selector';
+
+module.exports = React.createClass( {
```

Comment:
Regarding both your annotations: I actually had to refrain myself from using `export` and `class` and stateless components, as the few Calypso files I opened to understand Calypso's conventions didn't use them!
I'll update the files as soon as possible!